`test_scale_intensity_range_percentiles` error

**Describe the bug**
a frequent error for the recent unit testing, e.g. the log here: https://github.com/Project-MONAI/MONAI/actions/runs/1674183498
```
======================================================================
FAIL: test_channel_wise (tests.test_scale_intensity_range_percentiles.TestScaleIntensityRangePercentiles)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/test_scale_intensity_range_percentiles.py", line 86, in test_channel_wise
    assert_allclose(result, p(expected), rtol=1e-4)
  File "/home/jenkins/agent/workspace/MONAI-premerge/monai/tests/utils.py", line 92, in assert_allclose
    np.testing.assert_allclose(actual, desired, *args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/numpy/testing/_private/utils.py", line 1531, in assert_allclose
    verbose=verbose, header=header, equal_nan=equal_nan)
  File "/usr/local/lib/python3.7/dist-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
    raise AssertionError(msg)
AssertionError:
Not equal to tolerance rtol=0.0001, atol=0

Mismatched elements: 496 / 8192 (6.05%)
Max absolute difference: 255.
Max relative difference: 1.008
 x: array([[[0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],
        [0., 0., 0., ..., 0., 0., 0.],...
 y: array([[[0, 0, 0, ..., 0, 0, 0],
        [0, 0, 0, ..., 0, 0, 0],
        [0, 0, 0, ..., 0, 0, 0],...

----------------------------------------------------------------------
```

```
MONAI version: 0+unknown
Numpy version: 1.21.5
Pytorch version: 1.8.1+cu102
MONAI flags: HAS_EXT = True, USE_COMPILED = True
MONAI rev id: None

Optional dependencies:
Pytorch Ignite version: 0.4.7
Nibabel version: 3.2.1
scikit-image version: 0.19.1
Pillow version: 9.0.0
Tensorboard version: 2.7.0
gdown version: 4.2.0
TorchVision version: 0.9.1+cu102
tqdm version: 4.62.3
lmdb version: 1.3.0
psutil version: 5.9.0
pandas version: 1.3.5
einops version: 0.3.2
transformers version: 4.15.0
mlflow version: 1.22.0
```

this is still an issue
```py
======================================================================
FAIL: test_channel_wise (tests.test_scale_intensity_range_percentilesd.TestScaleIntensityRangePercentilesd)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/MONAI/MONAI/tests/test_scale_intensity_range_percentilesd.py", line 95, in test_channel_wise
    assert_allclose(scaler(data)["img"], p(expected), rtol=1e-4)
  File "/home/runner/work/MONAI/MONAI/tests/utils.py", line 93, in assert_allclose
    np.testing.assert_allclose(actual, desired, *args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 1530, in assert_allclose
    assert_array_compare(compare, actual, desired, err_msg=str(err_msg),
  File "/opt/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/numpy/testing/_private/utils.py", line 844, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=0.0001, atol=0

Mismatched elements: 853 / 8192 (10.4%)
Max absolute difference: 117
Max relative difference: 0.86029412
 x: array([[[[0, 0, 0, ..., 0, 0, 0],
         [0, 0, 0, ..., 0, 0, 0],
         [0, 0, 0, ..., 0, 0, 0],...
 y: array([[[[0, 0, 0, ..., 0, 0, 0],
         [0, 0, 0, ..., 0, 0, 0],
         [0, 0, 0, ..., 0, 0, 0],...

----------------------------------------------------------------------
```

```
MONAI version: 0+untagged.1.g32516e7
Numpy version: 1.22.1
Pytorch version: 1.6.0
```

--- a/tests/test_scale_intensity.py
+++ b/tests/test_scale_intensity.py
@@ -58,7 +58,7 @@ class TestScaleIntensity(NumpyImageTestCase2D):
     def test_channel_wise(self):
         for p in TEST_NDARRAYS:
             scaler = ScaleIntensity(minv=1.0, maxv=2.0, channel_wise=True)
-            data = p(self.imt)
+            data = p(np.tile(self.imt, (3, 1, 1, 1)))
             result = scaler(data)
             mina = self.imt.min()
             maxa = self.imt.max()

--- a/tests/test_scale_intensity_range_percentiles.py
+++ b/tests/test_scale_intensity_range_percentiles.py
@@ -66,7 +66,7 @@ class TestScaleIntensityRangePercentiles(NumpyImageTestCase2D):
         self.assertRaises(ValueError, ScaleIntensityRangePercentiles, lower=30, upper=900, b_min=0, b_max=255)
 
     def test_channel_wise(self):
-        img = self.imt[0]
+        img = np.tile(self.imt, (3, 1, 1, 1))
         lower = 10
         upper = 99
         b_min = 0
@@ -78,8 +78,8 @@ class TestScaleIntensityRangePercentiles(NumpyImageTestCase2D):
         for c in img:
             a_min = np.percentile(c, lower)
             a_max = np.percentile(c, upper)
-            expected.append(((c - a_min) / (a_max - a_min)) * (b_max - b_min) + b_min)
-        expected = np.stack(expected).astype(np.uint8)
+            expected.append((((c - a_min) / (a_max - a_min)) * (b_max - b_min) + b_min).astype(np.uint8))
+        expected = np.stack(expected)
 
         for p in TEST_NDARRAYS:
             result = scaler(p(img))

--- a/tests/test_scale_intensity_range_percentilesd.py
+++ b/tests/test_scale_intensity_range_percentilesd.py
@@ -75,7 +75,7 @@ class TestScaleIntensityRangePercentilesd(NumpyImageTestCase2D):
             s(self.imt)
 
     def test_channel_wise(self):
-        img = self.imt
+        img = np.tile(self.imt, (3, 1, 1, 1))
         lower = 10
         upper = 99
         b_min = 0

